src/components/Categories.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
```

Comment:
Double check your spelling here. The file name and the function name are different.